client/lib/purchases/index.js

```diff
@@ -71,6 +73,15 @@ function getName( purchase ) {
 	return purchase.productName;
 }
 
+function getFancyName( purchase ) {
+	if ( isJetpackProduct( purchase ) ) {
+		return JETPACK_PRODUCT_DISPLAY_NAMES[ purchase.productSlug ]
```

Comment:
We don't need this ternary here. This entire function could be simplified to something like:

```
function getFancyName( purchase ) {
	if ( isJetpackProduct( purchase ) && JETPACK_PRODUCT_DISPLAY_NAMES[ purchase.productSlug ] ) {
		return JETPACK_PRODUCT_DISPLAY_NAMES[ purchase.productSlug ];
	}
	return getName( purchase );
}
```